Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM ubuntu:18.04
+
+# Install required packages and remove the apt packages cache when done.
+
+RUN apt-get update && \
+    apt-get upgrade -y && \
+    apt-get install -y \
+        git \
+        python3 \
+        python3-dev \
+        python3-setuptools \
+        python3-pip \
+        nginx \
+        cron \
+        supervisor && \
+        pip3 install -U pip setuptools && \
+    rm -rf /var/lib/apt/lists/*
+
+# install uwsgi now because it takes a little while
+RUN pip3 install uwsgi
+
+# setup all the configfiles
+RUN echo "daemon off;" >> /etc/nginx/nginx.conf
+COPY config/nginx.conf /etc/nginx/sites-available/default
+COPY config/supervisor.conf /etc/supervisor/conf.d/
+
+# COPY requirements.txt and RUN pip install BEFORE adding the rest of your code, this will cause Docker's caching mechanism
+# to prevent re-installing (all your) dependencies when you made a change a line or two in your app.
+
+COPY app/requirements.txt /home/docker/code/app/
```

Comment:
We can really just copy to `/code` or `/src` or `/app` instead of `/home/docker/code/app/`, right?

Reply:
Yes. Does `/home/code` sound better?

Reply:
`/home` may confuse one with the home directory already in ubuntu. You can simply go with `/code`, then the Django files will actually reside in `/code/app`.

Reply:
Okay, thanks. I will change the directory and then we can probably merge this one!